docs/operation/cluster_and_service_configuration.rst

```diff
@@ -10,9 +10,58 @@ Managing Cluster and Services Configurations
 
 Newly deployed **MetalK8s** clusters come with chosen default values for most
 Cluster services. These default values are transparent to Admin users
-and can be customized at any point in time given that Admins follow the
+and can be customized at any point in time given that Administrators follow the
 documented procedure to the later.
 
+Managing default runtime Service Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+MetalK8s addons (Alertmanager, Dex, Grafana and Prometheus) ships with default
+runtime service configurations required for basic service deployment.
+Find below an exhaustive list of available default Service Configurations
+deployed in a MetalK8s cluster:
+
+Alertmanager
+""""""""""""
+The following base Alertmanager configurations are required for starting up
+Alertmanager within a MetalK8s cluster.
+To configure Alertmanager, override the default values of alertmanager.yaml
+file as specified below in it's corresponding Cluster and Service ConfigMap
+(metalk8s-alertmanager-config).
+This document below describes parameters that you can set.
```

Comment:
It's not a list of parameters that can be set, but just a default basic configuration, user could add some configuration that are not describe in this file, at least for Alertmanager.